src/components/GoodsList/GoodsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './GoodsList.scss';
+
+export const GoodsList = ({
+  goodsList,
+  isReverse,
+  sortBy,
```

Comment:
is better to write independent components, "goodsList" should not know reversed array or not, sorted or not, it just renders it, you can pass already prepared array into GoodsList component